e2e/README.md

```diff
@@ -0,0 +1,96 @@
+# e2e Testing
+
+This document captures information specific to e2e testing of Open MCT. For general information about testing, please see [testing section](link to testing section).
+
+## Overview
+
+e2e testing, sometimes known as "end to end" testing is an industry standard approach to automating the testing of Web UIs like Open MCT. Broadly speaking, e2e tests differentiate themselves in that they err on the side of replicating real user interaction over executing raw javascript functions like unit tests. 
+
+Historically, the abstraction necessary to replicate real user behavior meant:
+- that e2e frameworks were not capable of validing codepaths which were only reachable by unit tests
+- were expensive to run due to the need to instantiate a real browser
```

Comment:
if you're flowing in from "meant:" I'd recommend adding "e2e tests were expensive" in the beginning here.